ais/api/views.py

```diff
@@ -395,18 +477,14 @@ def search_view(query):
                 'include_units' in request.args,
                 is_range=high_num is not None,
                 is_unit=unit_type is not None) \
-                .exclude_non_opa('opa_only' in request.args)
+                .exclude_non_opa('opa_only' in request.args) \
```

Comment:
I made search route to opa_account_num view, intersection view and dor_parcel view for regmap. Currently the addresses endpoint supports batch queries. I want to make search support batch, but  I think some refactoring of the serializer design will be needed to allow serialization of different types per page. How do you feel about merging with how it is now and then revisiting the variable view call which looks nice?